node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -429,7 +430,7 @@ func (o OsDeviceConnectivityHelperGeneric) GetWwnByScsiInq(dev string) (string,
 	args := []string{"-p", "0x83", dev}
 	// add timeout in case the call never comes back.
 	logger.Debugf("Calling [%s] with timeout", sgInqCmd)
-	outputBytes, err := o.Executer.ExecuteWithTimeout(TimeOutSgInqCmd, sgInqCmd, args)
+	outputBytes, err := o.Executer.ExecuteWithTimeout(TimeOutSgInqCmd, sgInqCmd, args, true)
```

Comment:
could we have a variable for the boolean in each caller?

Reply:
since we usually want `true`, consider adding wrapper functions (e.g. `ExecuteWithTimeoutSilently`), so most callers won't have to deal with this

Reply:
didnt understand the first comment
done the second